UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -287,6 +293,13 @@ private void Update()
 			stationTime = stationTime.AddSeconds(Time.deltaTime);
 			roundTimer.text = stationTime.ToString("HH:mm");
 		}
+
+		timeElapsedQueueCheckServer += Time.deltaTime;
```

Comment:
this seems like it should be in update,  currently it is in private void Awake()

Reply:
It is in the update bod :p Just github view collapsed it.

Reply:
![image](https://user-images.githubusercontent.com/56727168/83813363-192ea480-a6b5-11ea-96ec-f9156dc0b4cc.png)